### ISSUE TITLE
Error decompressing go1.16.10.linux-amd64.tar.gz

### ISSUE THREAD

**Issue by kwboyd-shopify** [heroku/heroku-buildpack-go#479]
Appears to be the same thing that was happening in #452.

We're seeing this failure:
```


-----> New Go Version, clearing old cache
--
  | -----> Installing go1.16.10


[2021-11-30T17:44:20Z] gzip: stdin: not in gzip format
[2021-11-30T17:44:20Z] tar: Child returned status 1
[2021-11-30T17:44:20Z] tar: Error is not recoverable: exiting now
[2021-11-30T17:44:20Z] 2021/11/30 17:44:20 exit status 1
```

**Comment by edmorley** [comment]
@kwboyd-shopify Hi! Sorry for the disruption. The missing binaries for the new Go release have been published as of v159 (#480), which should resolve the error you were seeing.

**Comment by kwboyd-shopify** [comment]
Thank you @edmorley!
